less/about/siteDetails.less

```diff
@@ -82,6 +82,9 @@ body {
       font-size: 16px;
       margin-bottom: 12px;
       padding-left: @aboutPageSectionPadding;
+      width: 400px;
+      display: flex;
+      justify-content: space-between;
```

Comment:
Sorry, I didn't put the icon aligned. What I meant is 
```
<div className='title'>
  <span className='sectionTitle' data-l10n-id='versionInformation' />
   <span className='fa fa-clipboard' title='Copy password to clipboard' onClick={this.onCopy} />
 </div>
```
We wanna keep the original attribute for version information like orange color.
So we can put the attributes of `display` and `justify-content` into `.title`
And as @luixxiul suggest, we can add `margin-right` to `.fa-clipboard`